packages/fx-core/src/plugins/resource/apiconnector/sampleHandler.ts

```diff
@@ -42,14 +38,18 @@ export class SampleHandler {
         config: config,
         capitalName: config.APIName.toUpperCase(),
       };
-      const codeFile = compileHandlebarsTemplateString(templateString, context);
       const codeFileName: string = config.APIName + "." + fileSuffix;
-      await fs.writeFile(path.join(this.projectRoot, this.component, codeFileName), codeFile);
+      const codeFilePath = path.join(this.projectRoot, this.component, codeFileName);
+      if (await fs.pathExists(codeFilePath)) {
+        await fs.remove(codeFilePath);
```

Comment:
as discussed offline, we will return error if file already exists. will you have another PR to update the behavior?

Reply:
We will handle the existing file case in another PR.